README.md

```diff
@@ -19,7 +19,7 @@ Security Warnings
 -----------------
 
 See important security warnings in
-[doc/security-warnings.md](doc/security-warnings.md).
+[Security Information](https://z.cash/support/security/).
```

Comment:
`See important security warnings on the [Security Information page]()`.

Reply:
Done